src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Http2.cs

```diff
@@ -1598,19 +1598,36 @@ public async Task Http2GetAsync_TrailigPseudo_Throw()
         }
 
         [Fact]
-        [ActiveIssue(38708)]
         public async Task InboundWindowSize_Exceeded_Throw()
         {
             var semaphore = new SemaphoreSlim(0);
 
             await Http2LoopbackServer.CreateClientAndServerAsync(
                 async uri =>
                 {
-                    using HttpClient client = CreateHttpClient();
-                    using HttpResponseMessage response = await client.GetAsync(uri, HttpCompletionOption.ResponseHeadersRead);
+                    // An exception will be thrown by either GetAsync or ReadAsStringAsync once
+                    // the inbound window size has been exceeded. Which one depends on how quickly
+                    // ProcessIncomingFramesAsync() can read data off the socket.
+                    Exception requestException = await Assert.ThrowsAsync<HttpRequestException>(async () =>
```

Comment:
Nit: seems like the type here should be HttpRequestException, i.e. `HttpRequestException requestException = ...`